### ISSUE TITLE
IE 10 Does not work

### ISSUE THREAD

**Issue by lifenautjoe** [OpenbookOrg/openbook-org-www#51]
The website is not working on IE 10. 

Debug the issue, fix or alternatively create an unsupported page to send IE 10 and below users to.

**Comment by metaloha** [comment]
Is this necessary? It's generally accepted that versions of IE before 11 are really, really insecure and bad for your computer. Along with having maybe 0.5% of the market (including IE9 and IE8), I wouldn't think this is worth the effort to do.

If you absolutely need to use IE, use a version that Microsoft is still kind of supporting - IE 11.

**Comment by lifenautjoe** [comment]
Agreed. Let's for an unsupported page which instructs users to update their browser then.

The current case is that the website is just broken with no explanation.

**Comment by duichwer** [comment]
I'm not sure if this solves the issue, but when I inspect the HTML, it seems like there are two <head> groups. Modern browser display the website anyway, but this is still an error. I haven't worked with vue yet, but maybe this is caused by the missing closing <head> tag inside the index.html file.
